polyfills/Number/MIN_SAFE_INTEGER/config.json

```diff
@@ -0,0 +1,22 @@
+{
+	"aliases": [
+		"es6"
+	],
+	"browsers": {
+		"ie": "<12",
+		"ie_mob" : "*",
+		"firefox": "<31",
+		"firefox_mob":"<32",
+		"chrome":"<34",
+		"android": "<5",
+		"safari": "<9",
+		"ios_saf": "<9",
+		"opera": "*",
+		"op_mini": "*",
+		"op_mob": "*",
+		"bb": "*"
```

Comment:
Not needed, see -- https://github.com/Financial-Times/polyfill-service/pull/885#discussion_r79589886